script/setup

```diff
@@ -15,6 +15,10 @@ echo "==> Initializing and seeding database…"
 docker-compose exec app python manage.py migrate
 docker-compose exec app python manage.py seed
 
+echo 
+echo "==> Initialize static assets"
+docker-compose exec app python manage.py collectstatic
+
```

Comment:
This should probably go in update and server

Does this need to be re-run after code/branch changes? Does it need to be run once before starting a server?

Reply:
Should have clarified more. So this effects assets in the /admin and /swagger section of the django backend. 

Maybe a better way to handle this is to run `collectstatic` while building the container and dump the assets in. Thoughts?

Reply:
Decided this was probably better in the server and update scripts.